.github/workflows/CODE_SCANNING.yml

```diff
@@ -1,4 +1,4 @@
-name: "Code Scanning"
+name: CODE_SCANNING
```

Comment:
see [this](https://github.com/bpmn-io/moddle-xml/pull/76#discussion_r1825867036) comment

Reply:
I think we're fine, cf. https://github.com/camunda/camunda-modeler/pull/4657#issuecomment-2451970785.